src/Makefile.am

```diff
@@ -121,11 +121,14 @@ libsphde_la_SOURCES = \
 	sphlockfreeheap.cpp \
 	sphlflogger.cpp \
 	sphlogportal.cpp \
-	sphmultipcqueue.cpp \
 	sphsinglepcqueue.cpp \
 	sasallocpriv.h \
 	sphcontextpriv.h
 
+if HTM
+libsphde_la_SOURCES += sphmultipcqueue.cpp
```

Comment:
So that would prevent that file from being installed for use?  I suppose that's fine.  Is that change something you'd like me to make/commit?